Source/WebCore/html/PDFDocument.cpp

```diff
@@ -207,16 +207,16 @@ void PDFDocument::postMessageToIframe(const String& name, JSC::JSObject* data)
 
 void PDFDocument::sendPDFArrayBuffer()
 {
-    auto arrayBuffer = loader()->mainResourceData()->tryCreateArrayBuffer();
-    if (!arrayBuffer) {
-        ASSERT_NOT_REACHED();
-        return;
+    if (auto* documentLoader = loader()) {
```

Comment:
Should we consider ASSERT() instead for the loader?

Reply:
Possibly! I had trouble deciding because (as per the commit message) I don't know whether the loader is guaranteed to be non-null or not. I started with this conditional, then changed it to an assert, then got cold feet and changed back to a conditional before uploading. `Document::loader` definitely sometimes returns nullptr, but I'm not sure if that can happen for a PDFDocument that is already being loaded?

Reply:
Any preference here?

Reply:
I'd prefer an ASSERT here